src/main/java/cz/brmlab/yodaqa/io/web/WebInterface.java

```diff
@@ -47,13 +47,19 @@ public void run() {
 			@Override
 			public Object handle(Request request, Response response) {
 				String id = Integer.toString(idgen.nextInt(Integer.MAX_VALUE));
-				String text = request.queryParams("text");
+				String text=request.queryMap("question").value();
```

Comment:
I would swear that queryParams returned null every time. This is why I changed it to queryMap. However it works correctly. 
I also changed "question" back to "text". I don't know, what I was thinking.